app/models/search_builder.rb

```diff
@@ -96,6 +96,7 @@ def process_begins_with(value, op)
   end
 
   def process_is(value, op)
+    return nil if value.nil?
     return value if value.match(/"/) rescue true
```

Comment:
Maybe this should be
```suggestion
    return if value.blank?
    return value if value.match(/"/)
```

Reply:
If this turns out to be the issue, then we should probably add a unit test for when `value=""` to `process_is` spec.

Reply:
I will see if the tests pass with this version too,. Seems like it would.